.github/workflows/static-binary.yaml

```diff
@@ -18,6 +18,8 @@ jobs:
     env:
       BUILD_DIR: build
       STATIC_BINARY_TARGET: ${{ github.event.client_payload.args == '' && 'vast' || github.event.client_payload.args }}
+      PUBLIC_GCS_BUCKET: tenzir-public-data
+      STATIC_BINARY_FOLDER: vast-static-builds
```

Comment:
These only need to be available to a single step; can we move them there?

Reply:
Moved the env vars.